charts/smokey/templates/workflows/smokey-loop.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: argoproj.io/v1alpha1
+kind: CronWorkflow
+metadata:
+  name: smokey-loop
+spec:
+  schedule: "*/3 9-17 * * 1-5"
+  timezone: Europe/London
+  concurrencyPolicy: "Replace"
+  startingDeadlineSeconds: 0
```

Comment:
I don't think startingDeadlineSeconds is worth specifying here - the default is already 0 and it doesn't seem like something worth calling the reader's attention to.
```suggestion
```

Reply:
will do, thanks